service/templates/service/reset_password_confirm.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+{% load static %}
+
+<html lang="en">
+
+<head>
+
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>Register</title>
+
+    <!-- Bootstrap Core CSS -->
+    <link rel="stylesheet" type="text/css" href="{% static 'service/bootstrap/css/bootstrap.min.css' %}"/>
+
+    <link rel="stylesheet" type="text/css" href="{% static 'service/homepage/font-awesome/css/font-awesome.min.css' %}">
+    <link rel="stylesheet" type="text/css" href="{% static 'service/bootstrap/css/bootstrap.min.css' %}"/>
+
+    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
+    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
+    <!--[if lt IE 9]>
+    <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
+    <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
+    <![endif]-->
+
+</head>
+<body>
+    <div class="container">
+        <div class="row">
+            <div class="col-md-12">
+                <h1>Reset Password</h1>
+                <hr/>
+                <form id="reset_password_confirm_put" action="/reset_password_confirm_put/" method="post">{% csrf_token %}
+                    <fieldset>
+                        <legend>Enter new password</legend>
+
+                        <input type="password" placeholder="Password" id="password" name="password" value="{{ password }}">
+                        <input type="password" placeholder="Confirm Password" id="confirm_password" required>
+                        <input type="hidden" name="uidb64" value={{ uidb64 }}>
+                        <input type="hidden" name="token" value={{ token }}>
+                        <button type="submit" class="pure-button pure-button-primary">Confirm</button>
+                    </fieldset>
+                </form>
+            </div>
+        </div>
+    </div>
+    <script>
+        var password = document.getElementById("password")
+            , confirm_password = document.getElementById("confirm_password");
+
+        function validatePassword(){
+            if(password.value != confirm_password.value) {
+                confirm_password.setCustomValidity("Passwords Don't Match");
+            } else {
+                confirm_password.setCustomValidity('');
+            }
+        }
+
+        password.onchange = validatePassword;
+        confirm_password.onkeyup = validatePassword;
+    </script>
```

Comment:
This is cool !
I think we can do this verification rather than backend verification for our registration page too. Later.